agent_plugin_builder/build_options.py

```diff
@@ -21,7 +21,7 @@ class AgentPluginBuildOptions(InfectionMonkeyBaseModel):
         str,
         Field(
             title="The name of the source directory.",
-            default=None,
+            default="src",
```

Comment:
Is this a sane default?

Reply:
Well, it's not completely random. I suppose we could just get rid of the default value here